packages/app/src/systems/Ecosystem/components/ProjectItem/ProjectItem.tsx

```diff
@@ -83,7 +83,7 @@ export const ProjectItem: ProjectItemComponent = ({
           </Box.Stack>
           <Box.Flex align="center" justify="space-between" wrap="wrap">
             <ButtonLink
-              variant="link"
+              as="a"
```

Comment:
why change this ? looks not related to this pr objective

Reply:
When bumping fuel-ui the `ButtonLink` component no longer accepted the variant prop.  Not sure if this is a bug or not.